cmd/convert.go

```diff
@@ -111,4 +112,6 @@ func init() {
 	convertCmd.Flags().UintVarP(&threshold, "batch-threshold", "", 500, "batch request idle time threshold (see example)")
 	convertCmd.Flags().BoolVarP(&nobatch, "no-batch", "", false, "don't generate batch calls")
 	convertCmd.Flags().BoolVarP(&enableChecks, "enable-status-code-checks", "", false, "add a check for each http status response")
+	convertCmd.Flags().BoolVarP(&returnOnFailedCheck, "return-on-failure", "", false, "return from iteration if we get an unexpected response code")
```

Comment:
Could we change "return-on-failure" to "return-on-failed-check" and "... unexpected response code" to "unexpected response status code"? (we should change the description for `enable-status-code-checks` from "add a check for each http status response" to "add a status code check for each HTTP response" as well).

Reply:
Sure. I was back and forth with the naming a little bit myself.

Will fix right away.